app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_save :likes_counter
```

Comment:
* Kindly fix this typo here instead of `after_save :likes_counter` you should call the method correctly `update_likes_counter`

Reply:
Ok